ecosystem/node-checker/src/configuration/create.rs

```diff
@@ -13,9 +13,17 @@ pub struct Create {
 
     #[clap(flatten)]
     output_args: OutputArgs,
+
+    // If set, skip config validation. Use with great care.
```

Comment:
Theoretically I want to make a config that uses a path to a mint that doesn't exist on my system, but I know will be there in prod. Definitely an edge case but I want it there just in case.